### DISCUSSION TITLE
With the shortage of Raspberry Pi at the moment, has anyone tried this on the Raspberry Pi Pico W before I go and purchase one ?

### DISCUSSION THREAD

**Discussion by frogsuk** [cmcgerty/Sofar2mqtt#33] [Ideas]
With the shortage of Raspberry Pi at the moment, has anyone tried this on the Raspberry Pi Pico W before I go and purchase one?
[https://www.tomshardware.com/how-to/raspberry-pi-pico-w-node-red](url)
[https://nodered.org/docs/getting-started/raspberrypi](url)

**Comment by cliffdude** [comment]
Why do you want a pi? This runs on a  esp8266 based board like the Wemos D1
mini if you want to follow Colin's instructions exactly or can be adapted
to other boards like Nodemcu lua etc...
Search aliexpress or you local electronics supplier.

A segunda, 10/10/2022, 12:18, frogsuk ***@***.***> escreveu:

**Reply by frogsuk** [reply]
I know and have installed on the Wemos D1 without any problems at all.
Red Node / Home Assistant which normally runs on Raspberry Pi and wondered what others may be used as an alternative as Pi’s, in general, are all backorder or overpriced.
So wondered if the Pi Pico W which is cheap and available may be another alternative?

**Reply by MartinJSa** [reply]
I'm running HA as a virtual machine appliance in Hyper-V (on a windows box).. works perfectly for me
